chart/templates/pgbouncer/pgbouncer-deployment.yaml

```diff
@@ -59,6 +59,7 @@ spec:
         checksum/pgbouncer-config-secret: {{ include (print $.Template.BasePath "/secrets/pgbouncer-config-secret.yaml") . | sha256sum }}
         checksum/pgbouncer-certificates-secret: {{ include (print $.Template.BasePath "/secrets/pgbouncer-certificates-secret.yaml") . | sha256sum }}
     spec:
+      priorityClassName: {{ .Values.pgbouncer.priorityClassName }}
```

Comment:
I meant all instances of it should be conditional, not just flower :)

Reply:
sure, not done yet, will make those changes.